modules/home/assets/js/icons/side-bar-main-image-icon.js

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
```

Comment:
Can we delete this file? 
It isn't used. 

I believe the content is coming from the external json.